envs/example/ci-allinone/group_vars/all.yml

```diff
@@ -0,0 +1,15 @@
+---
+stack_env: example_ci_allinone
+floating_ip: 172.16.0.100
+
+xtradb:
+  sst_auth_password: asdf
+
+percona:
+  replication: False
+
+ceilometer:
+  enabled: True
```

Comment:
I copied that from existing envs/example/ci/ folder. Let me remove ceilometer in my code.